plugins/flytekit-vscode/flytekitplugins/vscode/decorator.py

```diff
@@ -5,20 +5,44 @@
 import sys
 import tarfile
 import time
+from dataclasses import dataclass, field
 from functools import wraps
-from typing import Callable, Optional
+from typing import Callable, List, Optional
 
 import fsspec
 
 from flytekit.loggers import logger
 
-from .constants import (
-    DEFAULT_CODE_SERVER_DIR_NAME,
-    DEFAULT_CODE_SERVER_REMOTE_PATH,
-    DEFAULT_UP_SECONDS,
-    DOWNLOAD_DIR,
-    EXECUTABLE_NAME,
-)
+from .constants import (DEFAULT_CODE_SERVER_DIR_NAME,
+                        DEFAULT_CODE_SERVER_EXTENSIONS,
+                        DEFAULT_CODE_SERVER_REMOTE_PATH, DOWNLOAD_DIR,
+                        EXECUTABLE_NAME)
+
+# Default max idle seconds to terminate the vscode server
+HOURS_TO_SECONDS = 60 * 60
+MAX_IDLE_SECONDS = 10 * HOURS_TO_SECONDS  # 10 hours
+
+# Duration to pause the checking of the heartbeat file until the next one
+HEARTBEAT_CHECK_SECONDS = 60
+
+# The path is hardcoded by code-server
+# https://coder.com/docs/code-server/latest/FAQ#what-is-the-heartbeat-file
```

Comment:
I think you can also move these to constants.py

Reply:
Make sense thanks!